features/deduct.js

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
This looks like is causes an issue when loading features on startup. I think  deleteing the file altogether should be fine though

Reply:
just removed it